app/renderer/components/Session/ServerTabTestingbot.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { Form, Input, Row, Col } from 'antd';
+
+const FormItem = Form.Item;
+
+export default class ServerTabTestingbot extends Component {
+
+  render () {
+
+    const {server, setServerParam, t} = this.props;
+
+    return <Form>
+      <Row gutter={8}>
+        <Col span={12}>
+          <FormItem>
+            <Input id='testingbotKey' placeholder={process.env.TB_KEY ? 'Using data found in $TB_KEY' : 'your-username'} addonBefore={t('TestingBot Key')} value={server.testingbot.key} onChange={(e) => setServerParam('key', e.target.value)} />
```

Comment:
All strings must be ready for translation

Reply:
The placeholder needs to also have a translated string.

```js
const placeholder = process.env.TB_KEY ? t('Using data found in', {environmentVariable: 'SAUCE_USERNAME'}) : t('your-username');
return <Form>
      <Row gutter={8}>
        <Col span={12}>
          <FormItem>
            <Input id='testingbotKey' placeholder={placeholder} addonBefore={t('TestingBot Key')} value={server.testingbot.key} onChange={(e) => setServerParam('key', e.target.value)} />

```